common/governments/hfe_gov.txt

```diff
@@ -15,10 +17,156 @@ hfe_gov_egypt = {
 
 hfe_gov_blight = {
 	ruler_title = hfe_gov_blight_ruler
+	ruler_title_female = hfe_gov_blight_ruler
```

Comment:
I dont think this line is nessesary, game will use male title.

Reply:
Paradox changed it as of V2.6 . I know this because I noticed blank Rulers around that time. Do you ever, EVER think that I have a reason for adding anything, or do you think that I am that stupid?